packages/plugin-user/src/user.js

```diff
@@ -274,15 +274,20 @@ const User = SparkPlugin.extend({
         })
         .catch((err) => Promise.reject(new Error(`failed to set authorization`, err)))
       )
-      .then(() => this.request({
-        service: `atlas`,
-        resource: `users/activations`,
-        method: `POST`,
-        headers,
-        body: options,
-        shouldRefreshAccessToken,
-        requiresClientCredentials
-      }))
+      .then(() => {
+        if (options.spoofUserAgent) {
```

Comment:
Sure. Although, is the user-agent header set from this point in the code when run from the browser? Or does that happen in a lower level browser call?